### PR TITLE
WINUI-3339-Visual-Styles-Avatarview

### PR DESCRIPTION
**Pull request by MadhumithanJayapathi**
### Description
Addition of Visual Styles Page for Avatar view UserGuide(UG).

### CODE REVIEW

File: winui/Avatar-view/avatarview_images/winui_circle_visualstyle_avatarview.png
Comment (Manivannan-E) [inline winui/Avatar-view/avatarview_images/winui_circle_visualstyle_avatarview.png:1]: @MadhumithanJayapathi Change the image as we discussed and change the image name as winui_circle_style_avatarview

File: winui/Avatar-view/visual-styles.md [hunk 1]
```diff
@@ -1,10 +1,404 @@
...
+                <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                    <ColumnDefinition Width="*"/>
+                </Grid.ColumnDefinitions>
+
+                <syncfusion:SfAvatarView AvatarShape="Circle" AvatarSize="ExtraLarge" Grid.Row="0" Grid.Column="4" Style="{StaticResource AvatarViewStyle}"/>
+                <TextBlock Text="ExtraLargeCircle" FontWeight="Bold" Grid.Row="1" HorizontalAlignment="Center" VerticalAlignment="Center" HorizontalTextAlignment="Center" TextAlignment="Center" Grid.Column="4" FontSize="10"/>
```
Comment (Manivannan-E) [inline winui/Avatar-view/visual-styles.md:62]: @MadhumithanJayapathi Remove the unwanted declared in the code example.

File: winui/Avatar-view/visual-styles.md [hunk 2]
```diff
@@ -1,10 +1,404 @@
...
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Visual Styles in WinUI AvatarView
\ No newline at end of file
+# Visual Styles in WinUI AvatarView
+
+`AvatarView` control allows for customization of visual styles by providing a range of built-in styles. You can configure the visual style of your AvatarView by using the `AvatarShape` property and select from the available options, such as:
```
Comment (Manivannan-E) [inline winui/Avatar-view/visual-styles.md:12]: @MadhumithanJayapathi The AvatarView control provides support for various built-in visual styles.

File: winui/Avatar-view/visual-styles.md [hunk 3]
```diff
@@ -1,10 +1,404 @@
...
+
+* Custom
+* Circle
+* Square
+
+## Custom
+
+Custom type enables extensive customization options such as adjusting the size, font size, corner radius, and other properties.
```
Comment (Manivannan-E) [inline winui/Avatar-view/visual-styles.md:20]: @MadhumithanJayapathi Custom type allows you to customize the AvatarView control with height, width, font and corner radius.

File: winui/Avatar-view/visual-styles.md [hunk 4]
```diff
@@ -1,10 +1,404 @@
...
+
+## Circle
+
+The `AvatarView` control allows you to set the size of the circle by utilizing the predefined `AvatarSize` values such as:
```
Comment (Manivannan-E) [inline winui/Avatar-view/visual-styles.md:24]: For the circle style, you can use the AvatarSize property to set the avatar's pre-defined circle size, which includes the following values:

Extra-large
Large
Medium
Small
Extra-small

File: winui/Avatar-view/avatarview_images/winui_circle_visualstyle_avatarview.png
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/avatarview_images/winui_circle_visualstyle_avatarview.png:1]: @Manivannan-E Changed the image as we discussed and changed the image name as winui_circle_style_avatarview

File: winui/Avatar-view/visual-styles.md [hunk 5]
```diff
@@ -1,10 +1,404 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: This section will provide an explanation of the various visual styles that are available in WinUI AvatarView.
```
[the remark this answers is not in the document]
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Changed the description as mentioned

File: winui/Avatar-view/visual-styles.md (hunk 1 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Removed the unwanted declared in the code example.

File: winui/Avatar-view/visual-styles.md (hunk 2 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Changed the content as mentioned

File: winui/Avatar-view/visual-styles.md (hunk 3 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Changed the content of custom as mentioned.

File: winui/Avatar-view/visual-styles.md (hunk 4 above)
Comment (MadhumithanJayapathi) [reply winui/Avatar-view/visual-styles.md]: @Manivannan-E Changed the content as mentioned.

File: winui/Avatar-view/visual-styles.md [hunk 6]
```diff
@@ -1,10 +1,364 @@
 ---
 layout: post
 title: Visual Styles in WinUI AvatarView control | Syncfusion
-description: This section will explain about the different types of visual styles that exists in WinUI AvatarView.
+description: Learn here all about the different visual styles that are available in the WinUI AvatarView.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:4]: Description: Learn about the different visual styles available in the WinUI AvatarView.

File: winui/Avatar-view/visual-styles.md [hunk 7]
```diff
@@ -1,10 +1,364 @@
...
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Visual Styles in WinUI AvatarView
\ No newline at end of file
+# Visual Styles in WinUI AvatarView
+
+The `AvatarView` control provides support for various built-in visual styles. You can configure the visual style of your AvatarView by using the `AvatarShape` property and select from the available options, such as:
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:12]: The `AvatarView` control provides support for various built-in visual styles. Configure the visual style of your AvatarView by using the `AvatarShape` property and selecting from the available options, such as:

File: winui/Avatar-view/visual-styles.md [hunk 8]
```diff
@@ -1,10 +1,364 @@
...
+
+* Custom
+* Circle
+* Square
+
+## Custom
+
+Custom type allows you to customize the AvatarView control with height, width, font and corner radius.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:20]: The custom type allows you to customize the AvatarView control with height, width, font, and corner radius.

File: winui/Avatar-view/visual-styles.md [hunk 9]
```diff
@@ -1,10 +1,364 @@
...
+
+## Circle
+
+For the circle style, you can use the `AvatarSize` property to set the avatar's pre-defined circle size, which includes the following values:
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:24]: For the circle style, use the `AvatarSize` property to set the avatar's pre-defined circle size, which includes the following values.

File: winui/Avatar-view/visual-styles.md [hunk 10]
```diff
@@ -1,10 +1,364 @@
...
+
+* Extra-large
+* Large
+* Medium
+* Small
+* Extra-small
+
+The following code sample illustrates how to set the visual styles of AvatarView to `Circle`.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:32]: The following code sample illustrates how to set the visual styles of AvatarView to `Circle.`

File: winui/Avatar-view/visual-styles.md [hunk 11]
```diff
@@ -1,10 +1,364 @@
...
+stack.Children.Add(mainGrid);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with circle visual styles](avatarview_images/winui_circle_style_avatarview.png)
+
+## Square
+
+For the square style, you can use the `AvatarSize` property to set the avatar's pre-defined square size, which includes the following values:
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:196]: For the square style, use the `AvatarSize` property to set the avatar's pre-defined square size, which includes the following values.

File: winui/Avatar-view/visual-styles.md [hunk 12]
```diff
@@ -1,10 +1,364 @@
...
+
+* Extra-large
+* Large
+* Medium
+* Small
+* Extra-small
+
+The following code sample illustrates how to set the visual styles of AvatarView to `Square`.
```
Comment (Chrispineagunja) [inline winui/Avatar-view/visual-styles.md:204]: The following code sample illustrates how to set the visual styles of AvatarView to `Square.`

**Pull request merged**